service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/AllocatedJudgeTemplate.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.fpl.model.notify;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
Don't think setter annotation is needed since fields are not final.

Reply:
Setter is needed, if the fields were final then the setter wouldn't be needed